src/Adapter/Order/CommandHandler/AddOrderFromBackOfficeHandler.php

```diff
@@ -164,4 +164,21 @@ private function assertAddressesAreNotDeleted(Cart $cart)
             ));
         }
     }
+
+    /**
+     * @param Cart $cart
+     *
+     * @return Country
+     *
+     * @throws \PrestaShopDatabaseException
+     * @throws \PrestaShopException
+     */
+    private function getTaxCountry(Cart $cart)
+    {
+        $taxAddressType = Configuration::get('PS_TAX_ADDRESS_TYPE');
+        $taxAddressId = $cart->{$taxAddressType};
```

Comment:
Since `$taxAddressType` can be set by configuration, you shoud check if the attribute is available  :thinking:

Reply:
Yep, good idea